android_jni/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
Please remove the second commit (Add Android JNI Wrapper) from this pull request. It should become a separate pull request.

I am not very familiar with GitHub's pull requests, so I don't know what is the best way to do this.

Reply:
Adding a commit to my fork seems to update this pull request as well. I was not expecting that. I have reverted that change now. This pull request only contains the one commit now.